cmd/cluster/core/create.go

```diff
@@ -1072,3 +1082,50 @@ func validateMgmtClusterAndNodePoolCPUArchitectures(ctx context.Context, opts *R
 
 	return nil
 }
+
+func validateVersion(ctx context.Context, opts *RawCreateOptions) error {
+	versionCLI := cliversion.GetRevision()
+
+	kubecfg, err := util.GetConfig()
+	if err != nil {
+		opts.Log.Info("failed to get Kubernetes config", "error", err)
+		return err
```

Comment:
I would _decorate_ the error with somethink like `fmt.Errorf("failed to get Kubernetes config: %w", err)`